configs/nri-prometheus-config.yml.sample

```diff
@@ -0,0 +1,20 @@
+integrations:
+  - name: com.newrelic.prometheus
+    exec:
+      - /var/db/newrelic-infra/newrelic-integrations/bin/nri-prometheus
+    config:
```

Comment:
wouldn't be better to use this?
```
- name: nri-prometheus
  config:
      ...
```
we don't need to specify the whole path, the agent will search for the executable in some known folders and we use the usual naming convention for the inventory 
https://docs.newrelic.com/docs/create-integrations/infrastructure-integrations-sdk/specifications/host-integrations-newer-configuration-format#inventory_source